tests/fixtures/issues/issue-900.php

```diff
@@ -22155,59 +22155,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-900-TL' => [
```

Comment:
I removed this test because it was getting picked up by the Generic file because I added the "Windows" platform.  I didn't think it was worth modifying the test since there's not really a windows version in this UA.